src/js/AutoAcceptRequestWatcher.js

```diff
@@ -0,0 +1,47 @@
+import Cookie from 'js-cookie';
+
+class AutoAcceptRequestWatcher {
+    constructor(
+        cookieConsentStore,
+        autoAcceptCookieConsentAfterRequestCount,
+        cookieSettings,
```

Comment:
Wil hier voor kijken met typescript om te bepalen wat de settings zijn. Gaat in dit geval op path, domain en name van de cookie